### PR TITLE
basic Spindicator

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Resolves #2065

**Comment by thisandagain** [comment]
Looks great and is working well when I set my device to 2G or 3G. `1000`ms feels like the right default.

File: www_src/pages/page/page.jsx [hunk 1]
```diff
@@ -298,6 +298,7 @@ var Page = React.createClass({
 
   load: function() {
     api({
+      spinOnLag: false,
```
Comment (thisandagain) [inline www_src/pages/page/page.jsx]: Why no spinner here?

**Comment by thisandagain** [comment]
R+ :spaghetti: :sheep:

**Pull request merged**